changelog.d/pr-190.md

```diff
@@ -0,0 +1,3 @@
+### 🚀 Enhancements and New Features
+
+- Add `--extra-inputs` to `containers-add`.  Fixes [#189](https://github.com/datalad/datalad-container/issues/189) via [PR #190](https://github.com/datalad/datalad-container/pull/190) (by [@nobodyinperson](https://github.com/nobodyinperson))
```

Comment:
feel welcome to expand here on how/when to use this feature, e.g. close to your use case.